f2/src/ConfirmationPage.js

```diff
@@ -141,7 +144,7 @@ export const ConfirmationPage = () => {
                   type: 'saveFormData',
                   data: { reportId },
                 })
-                let data = prepFormData(formData, reportId)
```

Comment:
I have no idea why `reportId` was in here, the `prepFormData` function doesn't use it!